tracer/test/test-applications/integrations/Samples.HotChocolate/Samples.HotChocolate.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  
+  <PropertyGroup>
+    <!-- override to remove not supported platforms -->
+    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
+    
+    <!-- Required to build multiple projects with the same Configuration|Platform, which is needed for the CI library/version matrix -->
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="HotChocolate.AspNetCore" Version="12.11.1" />
```

Comment:
This using a specific version - you need to do the `$(ApiVersion)` dance to make sure we test with multiple versions of the package 🙂